### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by ArmintasP**
- Refactored a method in `Utils.cs` to extend type `string`.
- Fixed logic mistake in method `CompareTo` from `Post.cs`.

### CODE REVIEW

**Review by PauliusLu — APPROVED** [review]
Nice to see some advanced c# features.

File: Karma/Models/Post.cs [hunk 1]
```diff
@@ -45,15 +45,15 @@ public static string GetPostTypeName(bool isDonation)
...
 
-        // By visibility and by date (decreasing order).
+        // By visibility and by date (descending order).
         public int CompareTo(Post other)
         {
             if (other.IsVisible == false && this.IsVisible == true)
-                return 1;
-            else if (other.IsVisible == true && this.IsVisible == false)
                 return -1;
+            else if (other.IsVisible == true && this.IsVisible == false)
```
Comment (baikauskaite) [inline Karma/Models/Post.cs:53]: Sorry that I didn't ask this in the last PR but I don't really understand why would we need to compare something by visibility? I guess posts that are not visible should not be shown at all, so there is no reason to implement sorting logic according to the visibility attribute?

**Review by baikauskaite — COMMENTED** [review]
Good default sorting feature, thank you.

File: Karma/Models/Post.cs (hunk 1 above)
Comment (baikauskaite) [reply Karma/Models/Post.cs:54]: Okay, sounds good.

**Pull request merged**